src/http-client.js

```diff
@@ -142,7 +142,8 @@ function parseHeaderValues(headers) {
   return parsedHeaders;
 }
 
-function buildRequest(input, init = {}) {
+function buildRequest(input, init) {
+  init || (init = {});
```

Comment:
Default parameters are only applied over undefined, and since the value is
not validated from user code before this point I think it's a good change.
On Sat, Feb 6, 2016 at 9:01 AM Mario A. Alvarez Garcia <
notifications@github.com> wrote:

> In src/http-client.js
> <https://github.com/aurelia/fetch-client/pull/37#discussion_r52104263>:
>
> > @@ -142,7 +142,8 @@ function parseHeaderValues(headers) {
> >    return parsedHeaders;
> >  }
> >
> > -function buildRequest(input, init = {}) {
> > +function buildRequest(input, init) {
> > +  init || (init = {});
>
> @vbauerster <https://github.com/vbauerster> Assigning default values like
> in the line below isn't more conventional? Besides, is more used throughout
> the framework. @bryanrsmith <https://github.com/bryanrsmith>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aurelia/fetch-client/pull/37/files#r52104263>.
>